CDEPS-interface/CMakeLists.txt

```
@@ -2,15 +2,21 @@
 if(CMAKE_Fortran_COMPILER_ID MATCHES "GNU")
   set(CMAKE_Fortran_FLAGS "-g -fbacktrace -ffree-line-length-none")
   if(${CMAKE_Fortran_COMPILER_VERSION} VERSION_GREATER_EQUAL 10)
-    set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -fallow-argument-mismatch -fallow-invalid-boz -DCPRGNU")
+    set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -fallow-argument-mismatch -fallow-invalid-boz")
+    #Set CPP defintions for the ufs/cdeps_share target library
+    set(CDEPS_SHARE_DEFS)
+    list(APPEND CDEPS_SHARE_DEFS "-DCPRGNU")
```

```suggestion
    list(APPEND CDEPS_SHARE_DEFS "CPRGNU")
```
`-D` is not necessary;`cmake` will ignore the `-D` and add in the makefile during the configuration.